client/src/components/RatingsReview.jsx

```diff
@@ -1,38 +1,201 @@
   useEffect(() => {
+    setPage(1)
+    setReviews([]);
     async function fetchAPI() {
-      var res = await Atelier.getReviews(13027);
+      var res = await Atelier.getReviews(props.id, 2, 1, sort);
+      //console.log('res: ' + JSON.stringify(res));
+      return res.results;
+    }
+    fetchAPI().then(res => {
       setReviews(res);
+    });
+  }, [sort]);
+
+  //runs when load more is clicked, fetch the next page of reviews
+  useEffect(() => {
+    async function fetchAPI() {
+      var res = await Atelier.getReviews(props.id, 2, page, sort);
+      return res.results;
+    }
+    if (page > 1) {
+      fetchAPI().then(res => {
+        setReviews(reviews.concat(res));
+      });
+    }
+  }, [page]);
+
+  //runs when the component loads, fetch the review metadata
+  useEffect(() => {
+    async function fetchAPI() {
+      var res = await Atelier.getMeta(props.id);
+      setMeta(res);
     }
     fetchAPI();
-  }, []);
-  return <div>{JSON.stringify(reviews)}</div>
+  }, [props.id]);
+
+  //handle the load more button, loads the next page
+  const handleLoad = () => {
+    setPage(page + 1);
+  };
+
+  //handle changing the sort dropdown, load new reviews
+  const handleChange = (e) => {
+    setSort(e.target.value)
+  };
+
+  //render only if there has been a review
+  if (Object.keys(reviews).length > 0) {
+    return (<div className='review-container'>
+      <div className='list-container'>
+        <form>
+          {/* sort drop down */}
+          <label>Sort by:</label>
+          <select value={sort} onChange={handleChange}>
+            <option value='helpful'>Helpful</option>
+            <option value='newest'>Newest</option>
+            <option value='relevant'>Relevance</option>
+          </select>
+        </form>
+          {/* main review table */}
+        <table className='review-table'>
+          <tbody>
+            <tr><td>Stars</td><td>Date</td><td>Summary</td><td>Body</td><td>Recommended</td><td>Name</td><td>Response</td><td>Helpfulness</td></tr>
+            {reviews.map(r => {
+              return <ReviewTile review={r} key={r.review_id} />
+            })}
+          </tbody>
+        </table>
+
+        <button onClick={handleLoad}>Load More</button> <button>Add a Review</button>
+      </div>
+      {/* product breakdown */}
+      <div className='breakdown'>
+        {Object.keys(meta).length > 0 ? <Breakdown data={meta} key={props.id} /> : <div></div>}
+      </div>
+    </div>)
+  } else {
+    return <div>Waiting for API response.</div>
+  }
+
 };
 
 const ReviewTile = (props) => {
+  var fields = ['date', 'summary', 'body', 'recommend', 'reviewer_name', 'response', 'helpfulness'];
+  var out = [];
+  for (var i = 0; i < fields.length; i++) {
+    out.push(JSON.stringify(props.review[fields[i]]));
+  }
+  //create row for each review tile
+  return <tr >
+    <td className='review-elem'><StarRating rating={props.review.rating} key={props.review.review_id} /></td>
```

Comment:
I would recommend breaking this line down into indentations at each prop passing
<Star 
  rating..
  key.. />